src/Admin/Views/Providers/Create.cshtml

```diff
@@ -3,14 +3,62 @@
     ViewData["Title"] = "Create Provider";
 }
 
+@section Scripts {
+    <script>
+        function toggleProviderTypeInfo(value) {
+            document.querySelectorAll('[id^="info-"]').forEach(el => { el.classList.add('d-none'); });
+            document.getElementById('info-' + value).classList.remove('d-none');
+        }
+    </script>
+}
+
 <h1>Create Provider</h1>
 
 <form method="post">
     <div asp-validation-summary="All" class="alert alert-danger"></div>
     
     <div class="form-group">
-        <label asp-for="OwnerEmail"></label>
-        <input type="text" class="form-control" asp-for="OwnerEmail">
+        <h2>Provider Type</h2>
+        <div class="form-check">
+            <input class="form-check-input" type="radio" asp-for="Type" name="Type" value="@((int)ProviderType.Msp)" id="radioMsp" onclick="toggleProviderTypeInfo(@((int)ProviderType.Msp))" checked>
+            <label class="form-check-label align-middle" for="radioMsp">
+                Managed Service Provider
+            </label>
+            <br/>
+            <label class="form-check-label small text-muted ml-3 align-top" for="radioMsp">
+                Access to clients organization
+            </label>
+        </div>
```

Comment:
Is there a better way to do this?

For example, I noticed:
* you set up display name attributes in the enum, but you're not using them here
* we use a helper to generate our select lists: https://github.com/bitwarden/server/blob/69ee26561596385b12a6a70beeac425a54bc7122/src/Admin/Views/Organizations/Edit.cshtml#L154-L156
* it looks like there might be a similar helper for radio buttons? https://www.tutorialsteacher.com/mvc/htmlhelper-radiobutton-radiobuttonfor

I don't know myself, but maybe have a look around, or let me know if you've already tried and it didn't work for some reason.

Reply:
Also, the `Display` attribute has a Description property that we can maybe use for the helper text: https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.displayattribute.description?view=net-6.0

I haven't done much with Razor pages, just looking for idiomatic ways to do things.

Reply:
For the Enums radio button list I couldn't use `Html.GetEnumSelectList` as that creates a DropDownList but I did manage to use `Html.RadioButtonFor`.

Excellent point on using the `DisplayNameAttribute` values. I'd never used it in the past in views so it required some figuring out but I'm glad with the result. I've replaced all static text to use the values from the attribute.

Reply:
I think you did a good job here!